pandas/core/groupby.py

```diff
@@ -2249,7 +2249,7 @@ def __init__(self, index, grouper=None, obj=None, name=None, level=None,
                 self.grouper = to_timedelta(self.grouper)
 
     def __repr__(self):
-        return 'Grouping(%s)' % self.name
+        return 'Grouping(%s)' % str(self.name)
```

Comment:
(This one is actually not related, so I can do it separately if needed.)

It failed if `self.name` was a tuple. For example:
```python
mi = pd.MultiIndex.from_arrays([list("AAB"), list("aba")])
df = pd.DataFrame([[1,2,3]], columns=mi)
gr = df.groupby(df[('A', 'a')])

gr.grouper.groupings
...
TypeError: not all arguments converted during string formatting
```